src/dapps/staked-dapp/TheStakedLayout.vue

```diff
@@ -40,12 +40,12 @@
         class="d-flex align-center justify-center mt-3 white--text"
         mandatory
         borderless
-        active-class="expandHeader font-weight-medium"
+        active-class="active-btn font-weight-medium"
         background-color="transparent"
       >
         <v-btn
           class="px-md-9 white--text text-transform--initial"
-          color="titlePrimary"
+          color="#00182c"
```

Comment:
Per russ, the inactive button is suppose to be "black" with opacity 24% (you just need to add a class for opacity and and change this to color="black")